maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/service/RepOrderService.java

```diff
@@ -38,10 +38,19 @@ public void updateAppDateCompleted(final UpdateAppDateCompleted updateAppDateCom
         log.info("update app date completed  - Transaction Processing - End");
     }
 
+    @Transactional
+    public void updateRepOrder(final UpdateRepOrder updateRepOrder) {
+        log.info("update rep order - Transaction Processing - Start");
+        RepOrderEntity repOrderEntity = repOrderImpl.find(updateRepOrder.getRepId());
+        repOrderMapper.updateRepOrderToRepOrderEntity(updateRepOrder, repOrderEntity);
+        repOrderImpl.updateRepOrder(repOrderEntity);
+        log.info("update rep order  - Transaction Processing - End");
+    }
+
     @Transactional
     public Boolean repOrderCountWithSentenceOrderDate(Integer repId) {
-        log.info("Get Rep order by appeal sentence order date is null - Transaction Processing - Start");
```

Comment:
I don't think it makes sense to include the word count in the method name if you're returning a boolean. Something like `hasRepOrderWithSentenceOrderDate` would be a lot more descriptive